api/routes/orders/controllers.js

```diff
@@ -15,9 +15,13 @@ class OrdersDetailController extends BaseDetailController {
 
   async post() {
     const _ordersService = this.app.getService('orders');
-    const { examinationIds } = this.request.body;
+    const { examinationIds, eventId } = this.request.body;
 
-    const order = await _ordersService.createOrder(examinationIds, this.request.user.id);
+    const order = await _ordersService.createOrder({
```

Comment:
Would need to check if event has expired or not based on `endDate`

Reply:
this has been already taken care inside `api/services/orders/index.js`